Sources/XMTPiOS/Contacts.swift

```diff
@@ -196,6 +204,7 @@ public actor Contacts {
 
 	// Whether or not we have sent invite/intro to this contact
 	var hasIntroduced: [String: Bool] = [:]
+    var latestUpdate: Date?
```

Comment:
Do you need this? Probably shouldn't have two sources of truth for the last time it was updated.